### PR TITLE
feat: introduce neutron-astroport-usdc_noble protocol

### PR DESCRIPTION
**Pull request by gmanev7**

### CODE REVIEW

File: protocol/contracts/lease/Cargo.toml [hunk 1]
```diff
@@ -8,9 +8,11 @@ license.workspace = true
 [package.metadata.cargo-each]
 combinations = [
     { tags = ["build", "$net", "$protocol"], always-on = ["contract", "$net", "$protocol"], include-rest = false, generics = { "$net" = "net", "$protocol" = "protocol" } },
-    { tags = ["ci", "$net", "$protocol"], feature-groups = ["net", "protocol", "contract-and-skel"], always-on = ["$net", "$protocol"], include-rest = true, generics = { "$net" = "net", "$protocol" = "protocol" } }
+    { tags = ["ci", "$net", "$protocol"], feature-groups = ["net", "protocol", "contract-and-skel", "testing"], always-on = ["$net", "$protocol"], include-rest = false, generics = { "$net" = "net", "$protocol" = "protocol" } },
+    { tags = ["build", "net_main", "neutron-astroport-usdc_noble"], always-on = ["net_main", "neutron-astroport-usdc_noble"], include-rest = false },
+    { tags = ["ci", "net_main", "neutron-astroport-usdc_noble"], always-on = ["net_main", "neutron-astroport-usdc_noble"], include-rest = false }
```
Comment (KirilMihaylov) [inline protocol/contracts/lease/Cargo.toml:13]: Let's keep then grouped and, preferably, in alphabetical order.

Comment (KirilMihaylov) [inline protocol/contracts/lease/Cargo.toml:13]: All occurrences.

```suggestion
    { tags = ["ci", "net_main", "neutron-astroport-usdc_noble"], feature-groups = ["testing"], always-on = ["net_main", "neutron-astroport-usdc_noble"], include-rest = false }
```

Comment (KirilMihaylov) [inline protocol/contracts/lease/Cargo.toml:13]: All occurrences.

Let's keep then grouped and, preferably, in alphabetical order.

File: protocol/packages/swap/Cargo.toml [hunk 2]
```diff
@@ -8,6 +8,7 @@ license.workspace = true
 [package.metadata.cargo-each]
 combinations = [
     { tags = ["ci", "net_dev", "net_test", "net_main", "neutron-astroport-usdc_axelar"], feature-groups = ["dex", "astroport-net"], always-on = ["astroport"], include-rest = true },
+    { tags = ["ci", "net_main", "neutron-astroport-usdc_noble"], always-on = ["main", "astroport"], include-rest = false },
```
Comment (KirilMihaylov) [inline protocol/packages/swap/Cargo.toml:11]: It's missing the `testing` feature as `include-rest` is set to `false`.
Please add a feature group for it identical, if not the same, as with the other packages.

**Review by KirilMihaylov — CHANGES_REQUESTED** [review]

File: protocol/contracts/lease/Cargo.toml (hunk 1 above)
Comment (gmanev7) [reply protocol/contracts/lease/Cargo.toml:13]: could you please check how to deal with oracle's combinations? There is no "testing" though "contract-and-testing" exists?

File: protocol/packages/swap/Cargo.toml (hunk 2 above)
Comment (gmanev7) [reply protocol/packages/swap/Cargo.toml:11]: nice catch!

File: protocol/contracts/lease/Cargo.toml (hunk 1 above)
Comment (gmanev7) [reply protocol/contracts/lease/Cargo.toml:13]: intuitively, I followed the other combination ... please check

**Review by KirilMihaylov — APPROVED** [review]
One minor suggestion; Otherwise looks good.

**Pull request merged**